app/assets/stylesheets/application.css

```diff
@@ -1,3 +1,97 @@
+
+body{
+  background-color: rgba(22, 136, 182, 0.85);
+}
+
+.user-stuff{
+  display: inline;
+}
+
+.user-stuff image_tag{
```

Comment:
you might have just meant `img` here? `image_tag` is a Rails helper that will generate an `img` in html.